unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/TheGraph/TheGraph.cs

```diff
@@ -12,7 +12,7 @@ public class TheGraph : ITheGraph
 {
     private const float DEFAULT_CACHE_TIME = 5 * 60;
     private const string LAND_SUBGRAPH_URL_ORG = "https://api.thegraph.com/subgraphs/name/decentraland/land-manager";
-    private const string LAND_SUBGRAPH_URL_ZONE = "https://api.thegraph.com/subgraphs/name/decentraland/land-manager-goerli";
+    private const string LAND_SUBGRAPH_URL_ZONE = "https://api.thegraph.com/subgraphs/name/decentraland/land-manager-sepolia";
```

Comment:
- This is used in `QueryLands`
- That is used in `FetchLandsFromOwner`

That last function is not used anywhere in the whole of Decentraland org so there should be no issue with changing it as is. https://github.com/search?q=org:decentraland%20FetchLandsFromOwner&type=code (Only used in the same but archived repo)